sources/tools/Stride.VisualStudio.PackageInstall/Stride.VisualStudio.PackageInstall.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net461</TargetFramework>
```

Comment:
It seems PackageInstall project only runs vsix as a process, so I decided to update to the latest version

Reply:
Did you test installing the VS extension after this change?

Reply:
Nope, I've only run the PackageInstall project, but yes. It should be tested.

Reply:
I'm not sure if I'm doing this correctly but after building the vsix plugin and copying into packageinstall location in runs visual studio plugin installer without issue 👍